env/.Python

```diff
@@ -0,0 +1 @@
+/Library/Frameworks/Python.framework/Versions/3.6/Python
```

Comment:
you should not commit the `env/` directory for your virtualenv. be sure to gitignore that folder!